articles/libraries/auth0-swift/index.md

```diff
@@ -55,9 +53,13 @@ To avoid specifying clientId & domain you can add a `Auth0.plist` file to your m
 </plist>
 ```
 
-### Authentication API
+## Using the Authentication API
+
+The Authentication API provides methods to authenticate the user against Auth0 server. 
+
+### Login with database connection
 
-#### Login with database connection
+Logging in with a database connection merely requires calling `login` with the user's username/email, password, and the name of the connection you wish to authenticate with. The response will be a Credentials object.
```

Comment:
Avoid the word merely. No comma needed after  `password,`
I would say something like using the 'Username-Password-Authentication' connection name, as it's not really like you have any options here for default database connection.